swarm/api/uri.go

```diff
@@ -69,7 +64,7 @@ func Parse(rawuri string) (*URI, error) {
 
 	// check the scheme is valid
 	switch uri.Scheme {
-	case "bzz", "bzz-raw", "bzz-immutable", "bzz-list", "bzz-hash", "bzzr", "bzzi", "bzz-resource":
+	case "bzz", "bzz-raw", "bzz-immutable", "bzz-list", "bzz-hash", "bzz-resource":
```

Comment:
it says further up that `bzz-hash` is deprecated, but it's still in the switch. Which is correct?

Reply:
I assumed that `bzz-hash` isn't deprecated since it is an integral part of usage of `bzz-immutable`, so i did not remove it.
Is it indeed deprecated?